src/pudl/extract/csv.py

```diff
@@ -0,0 +1,72 @@
+"""Extractor for CSV data."""
+from csv import DictReader
+from importlib import resources
+from zipfile import ZipFile
+
+import pandas as pd
+
+import pudl.logging_helpers
+
+logger = pudl.logging_helpers.get_logger(__name__)
+
+
+def open_csv_resource(dataset: str, base_filename: str) -> DictReader:
```

Comment:
This can be moved to an even more general space at some point but I didn't want to introduce another moving part in this PR.

Reply:
Yeah, I could see this moving to `pudl.helpers` or something but this is a fine place for it.